src/devices/video/hd44780.cpp

```diff
@@ -5,9 +5,113 @@
     Hitachi HD44780 LCD controller
 
     TODO:
-    - dump internal CGROM
-    - emulate osc pin, determine video timings and busy flag duration from it,
-      and if possible, remove m_busy_factor
+    - dump internal CGROM via decap
+    - finish emulating osc pin: determine video timings from it,
+       and if possible, remove m_busy_factor
+      The internal oscillator is known to be based on the value of the resistor
+       Rf, presumably (based on the HD44780U datasheet diagrams) with an
+       equation close to this:
+       For 5V power, Frequency in Hz = 1 / (2 * PI * Rf * 6.5pF)
+       For 3V power, Frequency in Hz = 1 / (2 * PI * Rf * 7.86pF)
+      The vast majority of devices use the internal oscillator with an Rf
+       resistor of 91KOhms, for ~270KHz clock speed.
+      However, a few Hitachi-made LCD modules use an Rf resistor of
+       200KOhms instead, for a ~122Khz clock speed, and it isn't clear
```

Comment:
It's with a small k, big H, so kOhm and kHz

Reply:
This should be fixed now.